src/components/BugBountyCards.tsx

```diff
@@ -1,154 +1,105 @@
+import { BaseHTMLAttributes } from "react"
 import { useTranslation } from "next-i18next"
-import {
-  Box,
-  type BoxProps,
-  Center,
-  type CenterProps,
-  Divider,
-  Flex,
-  type FlexProps,
-  Heading,
-  type HeadingProps,
-  type TextProps,
-  useToken,
-} from "@chakra-ui/react"
 
 import type { ChildOnlyProp, TranslationKey } from "@/lib/types"
 
-import { ButtonLink, type ButtonLinkProps } from "@/components/Buttons"
-import Text from "@/components/OldText"
+import { cn } from "@/lib/utils/cn"
+
+import { ButtonLink, ButtonLinkProps } from "./ui/buttons/Button"
+import { Center, Flex, Stack } from "./ui/flex"
+
+type FlexProps = BaseHTMLAttributes<HTMLDivElement>
 
 const CardRow = ({ children }: ChildOnlyProp) => (
-  <Flex justifyContent="space-between" my="16" mx="4" flexWrap="wrap">
-    {children}
-  </Flex>
+  <Flex className="mx-4 my-16 flex-wrap justify-between">{children}</Flex>
 )
 
 const SubmitBugBountyButton = ({ children, ...props }: ButtonLinkProps) => (
-  <ButtonLink m="4" href="https://forms.gle/Gnh4gzGh66Yc3V7G8" {...props}>
+  <ButtonLink
+    className="m-4"
+    href="https://forms.gle/Gnh4gzGh66Yc3V7G8"
+    {...props}
+  >
     {children}
   </ButtonLink>
 )
 
 const Card = ({ children, ...props }: FlexProps) => {
-  const tableBoxShadow = useToken("colors", "tableBoxShadow")
-
   return (
-    <Flex
-      flexDir="column"
-      flex={{ base: "1 1 412px", xl: "1 1 260px" }}
-      justifyContent="space-between"
-      bg="background.base"
-      borderRadius="2px"
-      borderRad
-      boxShadow={tableBoxShadow}
-      border="1px solid"
-      borderColor="border"
-      m="4"
-      _hover={{
-        borderRadius: "base",
-        boxShadow: "tableBoxHover",
-        background: "tableBackgroundHover",
-        transition: "transform 0.1s",
-        transform: "scale(1.02)",
-      }}
+    <Stack
+      className={cn(
+        "flex-[1_1_412px] gap-0 xl:flex-[1_1_216px]",
+        "m-4 justify-between bg-background",
+        "rounded-sm border border-[#e5e5e5] shadow-table-box dark:border-[#333]",
+        "hover:scale-[1.02] hover:rounded hover:bg-[#f2f2f2] hover:shadow-table-box-hover hover:transition-transform hover:duration-100 dark:hover:bg-[rgba(255,115,36,.013)]"
+      )}
       {...props}
     >
       {children}
-    </Flex>
+    </Stack>
   )
 }
 
 type LabelVariant = "low" | "medium" | "high" | "critical"
 
-type LabelProps = CenterProps & {
+type LabelProps = FlexProps & {
   variant: LabelVariant
 }
 
-const stylePropsByVariant = {
-  low: {
-    bg: "lowBug",
-    color: "black300",
-  },
-  medium: {
-    bg: "mediumBug",
-    color: "black300",
-  },
-  high: {
-    bg: "fail400",
-    color: "white",
-  },
-  critical: {
-    bg: "fail600",
-    color: "white",
-  },
+const classNameByVariant = {
+  low: "bg-[#ffe3d3] text-[#333]",
+  medium: "bg-[#ffab7b] text-[#333]",
+  high: "bg-[#c63333] text-white",
+  critical: "bg-[#930000] text-white",
```

Comment:
@nloureiro would you like to do something this these colors? or should we keep them for now?

<https://deploy-preview-14000--ethereumorg.netlify.app/en/bug-bounty/>
![image](https://github.com/user-attachments/assets/2e656f1f-d5b4-4dbe-ba7b-d9d9887c6880)

Reply:
Would personally suggest just updating to use our red palette at the very least (screenshots above):

```suggestion
  low: "bg-red-100 text-black",
  medium: "bg-red-300 text-black",
  high: "bg-red-700 text-white",
  critical: "bg-red-900 text-white",
```

<img width="1711" alt="image" src="https://github.com/user-attachments/assets/3042a5b8-2977-4127-b04b-d775682afb10">

<img width="1615" alt="image" src="https://github.com/user-attachments/assets/e787440f-d993-4798-b552-4bbca0ecc126">

cc: @nloureiro

Reply:
Like it. @nloureiro to confirm and approve